test/nuxeo-breadcrumb.test.js

```diff
@@ -0,0 +1,207 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { fakeServer, fixture, flush, html, isElementVisible, login } from '@nuxeo/testing-helpers';
+import '../elements/nuxeo-browser/nuxeo-breadcrumb.js';
+
+const document = {
+  'entity-type': 'document',
+  contextParameters: {
+    breadcrumb: {
+      entries: [
+        {
+          path: '/default-domain',
+          title: 'Domain',
+          type: 'Domain',
+          uid: '1',
+        },
+        {
+          path: '/default-domain/workspaces',
+          title: 'Workspaces',
+          type: 'WorkspaceRoot',
+          uid: '2',
+        },
+        {
+          path: '/default-domain/workspaces/my workspace',
+          title: 'my workspace',
+          type: 'Workspace',
+          uid: '3',
+        },
+        {
+          path: '/default-domain/workspaces/my workspace/folder 1',
+          title: 'folder 1',
+          type: 'Folder',
+          uid: '4',
+        },
+        {
+          path: '/default-domain/workspaces/my workspace/folder 1/folder 2',
+          title: 'folder 2',
+          type: 'Folder',
+          uid: '5',
+        },
+        {
+          path: '/default-domain/workspaces/my workspace/folder 1/folder 2/folder 3',
+          title: 'folder 3',
+          type: 'Folder',
+          uid: '6',
+        },
+        {
+          path: '/default-domain/workspaces/my workspace/folder 1/folder 2/folder 3/my file',
+          title: 'my file',
+          type: 'File',
+          uid: '7',
+        },
+      ],
+    },
+  },
+  path: '/default-domain/workspaces/my workspace/folder 1/folder 2/folder 3/my file',
+  title: 'my file',
+  type: 'File',
+  uid: '7',
+};
+
+// Mock router
+const router = {
+  browse: (path) => path.substring(1),
+};
+
+suite('nuxeo-breadcrumb', () => {
+  let server;
+
+  setup(async () => {
+    server = fakeServer.create();
+    await login();
```

Comment:
Login already returns a new fake server. I'd expect this to work as well:```
```suggestion
    server = await login();
```

Reply:
Indeed, that method is deprecated and you [should use MockClient](https://github.com/nuxeo/nuxeo-elements/blob/master/testing-helpers/test-helpers.js#L144-L147) instead.